src/chunk.js

```diff
@@ -404,6 +407,16 @@ class Chunks {
    * @private
    */
   executeAsap_(idleDeadline) {
+    // If we've spent over 5 millseconds executing the
+    // last instruction yeild back to the main thread.
+    // 5 milliseconds is a magic number.
+    if (
+      this.timeSinceLastExecution_ !== 0 &&
```

Comment:
Good catch. But I think it is better to avoid this special case by initializing `this.timeSinceLastExecution` with `Date.now()`

Reply:
Makes sense, applied the change.